woo-razorpay.php

```diff
@@ -1383,7 +1383,7 @@ public function orderArg1CC($data, $order)
                    $product_id = $product->get_parent_id();
                }
                $data['line_items'][$i]['product_id'] = (string)$product_id;
-			   $data['line_items'][$i]['weight'] = $product->get_weight()*1000*(int)$item->get_quantity();
+			   $data['line_items'][$i]['weight'] = $this->getWeightBasedOnStoreSettingsAndQuantity($product->get_weight(), $item->get_quantity());
```

Comment:
@Anantha-Perumal Have you tried [this](https://wp-kama.com/plugin/woocommerce/function/wc_get_weight) wooc default method which matches with our requirement? 
If this function is not working then we can go ahead with the custom function which you had written.

Reply:
tried it, it works refactored the code